tests/debuginfo/constant-ordering-prologue.rs

```diff
@@ -1,5 +1,7 @@
 //@ min-lldb-version: 310
 
+//@ only-x86_64-unknown-linux-gnu
```

Comment:
This is some newly introduced test? I think `ignore-aarch64-unknown-ferrocenecoretest` would be more appropriate? (I think this should run on Mac etc no?)

Reply:
Maybe I assumed that it should not because the test failed for aarch64-darwin.

Reply:
we have a couple more debuginfo tests currently ignored fwiw https://github.com/ferrocene/ferrocene/blob/eecd617bc6f3c693099a1ba900abb93ab9db9196/ferrocene/ignored-tests.toml#L62-L70